features/step_definitions/background_step.rb

```diff
@@ -14,6 +14,14 @@
   end
 end
 
+Given("the following product has been added to order") do |table|
+  table.hashes.each do |order_attributes|
+    @order = @user.orders.create
+    product = Product.find_by(name: order_attributes['name'])
+    @order.add(product, product.price)
```

Comment:
I don’t see a reason to use instance variables here. Local will do

Reply:
correctm fixed.